webpack.config.js

```diff
@@ -65,11 +65,28 @@ const webpackConfig = {
 			{
 				include: /node_modules\/tinymce/,
 				loader: 'imports?this=>window',
-			}
+			},
+			{
+				test: /\.scss$/,
+				loaders: [
+					'isomorphic-style-loader',
+					'css-loader?camelCase=dashes&importLoaders=1',
+					'sass-loader',
+					"webpack-append?@import 'assets/stylesheets/shared/component';",
```

Comment:
You can do this without an additional loader using `sass-loader`'s `data` query argument.

Example: https://github.com/WordPress/gutenberg/blob/935b958/webpack.config.js#L60